lib/model/katex.dart

```diff
@@ -209,11 +209,102 @@ class _KatexParser {
         debugHtmlNode: debugHtmlNode);
     }
 
+    if (element.className == 'vlist-t'
+        || element.className == 'vlist-t vlist-t2') {
+      final vlistT = element;
+      if (vlistT.nodes.isEmpty) throw _KatexHtmlParseError();
+      if (vlistT.attributes.containsKey('style')) throw _KatexHtmlParseError();
+
+      final hasTwoVlistR = vlistT.className == 'vlist-t vlist-t2';
```

Comment:
There's also a CSS rule for .vlist-t2 in the KaTeX CSS:
```css
    .vlist-t2 {
        margin-right: -2px;
    }
```

What's the net effect of that? On its face it looks like something we should have to take account of.

Reply:
This will need support for negative margins, so will add a commit in the next PR for this. (It will also need changes for supporting non-`em` values in KatexSpanStyles.)